man/systemd.network.xml

```diff
@@ -3279,7 +3288,10 @@ DHCP=ipv6</programlisting>
 Name=enp2s0
 
 [Network]
-IPv6PrefixDelegation=dhcpv6</programlisting>
+IPv6PrefixDelegation=dhcpv6
+
+[DHCPv6]
+AssignAcquiredDelegatedPrefixAddress=true</programlisting>
```

Comment:
doesn't matter much, but in all our examples so far we use "yes" rather than "true", even if both are accepted